lib/cosmos/tools/tlm_grapher/data_objects/housekeeping_data_object.rb

```diff
@@ -252,14 +252,10 @@ def process_packet(packet, count)
 
         # Prune Data
         prune_to_max_points_saved()
-      rescue Exception => error
```

Comment:
I don't know about this.  I don't want one bad data point we don't handle to bring down TlmGrapher...